tables-to-go.go

```diff
@@ -49,6 +49,7 @@ func NewCmdArgs() (args *CmdArgs) {
 	flag.Var(&args.Null, "null", "representation of NULL columns: sql.Null* (sql) or primitive pointers (native|primitive)")
 
 	flag.BoolVar(&args.NoInitialism, "no-initialism", args.NoInitialism, "disable the conversion to upper-case words in column names")
+	flag.StringVar(&args.CustomColumnRename, "ccr", args.CustomColumnRename, "add custom colum rename rules and ignore column renaming functions, skipping initialism and casing (format: providerId:ProviderID,123:abc)")
```

Comment:
Same thought as in the comment of the godoc of the parsing function, lets be super explicit here, since this is the helper/usage output for users of this tool. Maybe something along the lines

```
use custom-column-rules to overwrite column naming and initialism. Format ...
```

Can you double check the typo `colum` -> `column`?